### ISSUE TITLE
Can`t See GPU Queue In Lastest Version

### ISSUE THREAD

**Issue by jiangzuojiben** [google/agi#1191]
**Environment information:**
 - AGI version:2022-3.0.1
 - Host OS: Windows
 If tracing on Android:
 - Device model: Samsun Galaxy S10
 - Android Version:

![image](https://user-images.githubusercontent.com/38678895/187348734-6d23bccb-a17d-423c-9a08-b027a7f7946f.png)

**Comment by ttanatb** [comment]
Thanks for filing this issue, could you provide more information so we can try to triage what's going on? Could you provide `gapis.log` and `gapic.log` from the top-left menu `Help > Open Log Directory` along with the trace file (the `.perfetto` file) and the `adb logcat > log.txt` 

Also, is this device using an Adreno GPU? If so, it could be https://github.com/google/agi/issues/1113 which would cause GPU Counters track to be empty (but not sure about the GPU track or the Surface Flinger track)

**Comment by jiangzuojiben** [comment]
YES, this device use Adreno GPU, sorry about i can`t upload file now

**Comment by ttanatb** [comment]
It's likely that you're facing the same issue as #1113 which is an issue we're trying to resolve with Samsung. There was a security vulnerability earlier that Qualcomm patched in a way that was subpar. Basically, they gated access to GPU counters behind the file `/sys/class/kgsl/kgsl-3d0/perfcounter`. The newest dev release of AGI https://github.com/google/agi-dev-releases/releases/tag/v3.1.0-dev-20220830 tries to enable GPU counters, but different devices have different permission access requirements for this file.

Users have been reporting that for Samsung devices the access for this file is hidden behind root access. If you can afford to root your phone then you can try
```
adb root
adb shell "echo 1 > /sys/class/kgsl/kgsl-3d0/perfcounter"
```

but we understand that this is not an acceptable solution to lots of our users and are working to resolve this.

**Comment by ttanatb** [comment]
Following up on this @jiangzuojiben are you still running into the same issue? Other users have been reporting that they're no longer running into the Adreno GPU counter issue on their samsung devices.
